Makefile

```diff
@@ -1,35 +1,41 @@
-PHONY: all build test clean build-image push-image
+PHONY: all build test clean build-image deploy undeploy push-image run-es
 .DEFAULT_GOAL := all
 
-IMAGE_PREFIX ?= ctovena
+include .bingo/Variables.mk
+
+IMAGE_PREFIX ?= quay.io/openshift-logging
 IMAGE_TAG := 0.1
 ES_CONTAINER_NAME=elasticsearch
 ES_IMAGE_TAG=elasticsearch:6.8.12
 
 all: test build-image
 
-build:
-	go build -o logger -v main.go es_bulk_indexer.go
+lint:
+	bingo get golang.org/x/lint/golint
+	golint main.go
```

Comment:
`bingo get` is not need here. It is actually a single-shot command when you first introduce the tool in question, e.g. `golint`. Since you include `.bingo/Variables.mk` you can reference the tool like this without polluting your repo's `go.mod` an `go.sum` files:
```makefile
lint: $(GOLINT)
    $(GOLINT) main.go
```

Reply:
fixed
DONE